pluginbase/src/scopyjs.cpp

```diff
@@ -233,4 +233,56 @@ void ScopyJS::jsCategoryFilter(QLoggingCategory *category)
 	}
 }
 
+void ScopyJS::importFileContent(const QString &path)
+{
+	QTextStream out(stdout);
+	QFile file(path);
+	if(!file.open(QFile::ReadOnly)) {
+		out << "Unable to open the script file: " << path << Qt::endl;
+		return;
+	}
+	const QString scriptContent = getScriptContent(&file);
+	QJSValue val = ScopyJS::GetInstance()->engine()->evaluate(scriptContent, path);
+	if(val.isError()) {
+		out << "Exception:" << val.toString() << Qt::endl;
+	} else if(!val.isUndefined()) {
+		out << val.toString() << Qt::endl;
+	}
+
+	out.flush();
+}
+
+QJSValue ScopyJS::importFileAs(const QString &path)
```

Comment:
Do you have an example for this ?

Reply:
I think both functions serve the same purpose .. are both needed as one just discards the return value ? you can simply not use it.

Also you can rename this to "evaluateFile"